pkg/destroy/aws/aws.go

```diff
@@ -1825,6 +1825,41 @@ func deleteS3(session *session.Session, arn arn.ARN, logger logrus.FieldLogger)
 	if err != nil && !isBucketNotFound(err) {
 		return err
 	}
+
+	var lastError error
+	err = client.ListObjectVersionsPagesWithContext(aws.BackgroundContext(), &s3.ListObjectVersionsInput{
+		Bucket: aws.String(arn.Resource)}, func(page *s3.ListObjectVersionsOutput, lastPage bool) bool {
+		var deleteObjects []*s3.ObjectIdentifier
+		for _, deleteMarker := range page.DeleteMarkers {
+			deleteObjects = append(deleteObjects, &s3.ObjectIdentifier{
+				Key:       aws.String(*deleteMarker.Key),
+				VersionId: aws.String(*deleteMarker.VersionId),
+			})
+		}
+		for _, version := range page.Versions {
+			deleteObjects = append(deleteObjects, &s3.ObjectIdentifier{
+				Key:       aws.String(*version.Key),
+				VersionId: aws.String(*version.VersionId),
+			})
+		}
+		_, err := client.DeleteObjects(&s3.DeleteObjectsInput{
+			Bucket: aws.String(arn.Resource),
+			Delete: &s3.Delete{
+				Objects: deleteObjects,
+				Quiet:   aws.Bool(false),
```

Comment:
Hmm.. what does quiet do? We should have quiet true right? We don't want this generating noise?

Reply:
I've changed quiet to be true.